### ISSUE TITLE
Use Ansistrano for cleaner deployments

### ISSUE THREAD

**Issue by Matt-Yorkley** [openfoodfoundation/ofn-install#491]
As noted here: https://github.com/openfoodfoundation/ofn-install/pull/471#issuecomment-513268962

We can switch to Ansistrano, and it should improve our deployment system and clean up the repo a bit.

**Comment by mkllnk** [comment]
Yes, I think this would be a lot better than what we have at the moment.

**Comment by mkllnk** [comment]
Closing due to no activity.
